inst/templates/widget_js_new.txt

```diff
@@ -0,0 +1,19 @@
+HTMLWidgets.widget({
+
+  name: '%s',
+
+  type: 'output',
+
+  factory: function(el, width, height) {
+
+    return {
```

Comment:
Maybe also some one line comments on what each function is for (we do a lot of this in the rstudio templates and people seem to really appreciate it)